lib/ws_servers/api/handlers/strategy/strategy_manager.js

```diff
@@ -568,6 +568,7 @@ class StrategyManager {
       symbol,
       strategyId,
       strategyOptions,
+      strategyMapKey,
```

Comment:
`id` is the same as `strategyMapKey`. We can directly read from id instead of adding a new field.